flash/graph/classification/data.py

```diff
@@ -44,6 +50,121 @@ def from_datasets(
         transform_kwargs: Optional[Dict] = None,
         **data_module_kwargs,
     ) -> "GraphClassificationData":
+        """Load the :class:`~flash.graph.classification.data.GraphClassificationData` from PyTorch Dataset objects.
+
+        The Dataset objects should be one of the following:
+
+        * A PyTorch Dataset where the ``__getitem__`` returns a tuple: ``(PyTorch Geometric Data object, target)``
+        * A PyTorch Dataset where the ``__getitem__`` returns a dict:
+            ``{"input": PyTorch Geometric Data object, "target": target}``
```

Comment:
```suggestion
            ``{"input": PyTorch Geometric Data object, "target": label/class}``
```

Reply:
Will leave for now as the nature of the target would depend on the formatting being applied.